charts/radix-cluster-cleanup/templates/deployment.yaml

```
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "radix-cluster-cleanup.fullname" . }}
+  namespace: radix-cluster-cleanup
```

```suggestion
  namespace: {{ .Release.Namespace }}
```
Namespace should use the namespace for the helm release, same as ServiceAccount